src/components/ArticleView/AllArticles.jsx

```diff
@@ -0,0 +1,36 @@
+import ArticlePreviewCard from "./ArticlePreviewCard";
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+
+const AllArticles = () => {
+  const [articlesAll, setArticlesAll] = useState([]);
+  React.useEffect(() => {
+    axios
+      .get("https://nc-news-iweb.onrender.com/api/articles")
+      .then(({ data }) => {
+        const { articleData } = data;
+        setArticlesAll(articleData);
+      })
+      .catch((err) => {
+        Promise.reject(err);
+      });
+  }, []);
+
+  return (
+    <ul>
+      {articlesAll.map((article) => (
+        <ArticlePreviewCard
+          key={article.article_id}
```

Comment:
consider just sending through the entire article={article} as a single prop which can be destructured or use directly in this component rather than needing to declare each prop individually